inc/api/namespace.php

```diff
@@ -198,6 +198,11 @@ function get_graph_data( $start, $end, $resolution = '1 day', ?Filter $filter =
 	global $wpdb;
 
 	$query_where = [ '1=1' ];
+	$query_params = [
+		'param_blog_id' => get_current_blog_id(),
```

Comment:
This kind of leaks the REST API behaviour with the prefix included. I'd expect
```
Utils\clickhouse_query( 'SELECT * FROM analytics WHERE blog_id = {blog_id:string}', [
    'blog_id' => get_current_blog_id(),
] )
```

Suggest moving the prefix to the query function instead

Reply:
That was intentional, thought it might be useful.

Reply:
Yeah, looking at some of the options that's probably best handled more intentionally via a filter if at all. I'll do this for now.

Reply:
If we switched to eg a native Clickhouse TCP client driver, I'd expect the call interface of this function to stay the same, but the params wouldn't need to be prefixed

Reply:
Still uses the prefixes https://clickhouse.com/docs/en/interfaces/cli/#cli-queries-with-parameters